contrib/babelfishpg_common/sql/upgrades/babelfish_common_helper--4.4.0--4.5.0.sql

```diff
@@ -0,0 +1,24 @@
+------------------------------------------------------------------------------
+---- Include changes related to other datatypes except spatial types here ----
+------------------------------------------------------------------------------
+
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
+\echo Use "ALTER EXTENSION ""babelfishpg_common"" UPDATE TO "4.5.0"" to load this file. \quit
+
+SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+CREATE OR REPLACE FUNCTION  sys.smalldatetime_date_cmp(sys.SMALLDATETIME, date)
```

Comment:
run this script repeatedly to ensure that there is not any issue with the upgrade path

Reply:
This failed as exception handling was missing but added a exception handling block and tested locally and it works fine.